### PR TITLE
Add blog post (and template) for lab meeting report 18 July

### PR DESCRIPTION
**Pull request by KirstieJane**
Added a blog post for 18 July and a template for future meetings.

Let me know what you think of the format. Tagging @GeorgiaHCA for review, but anyone can chime in ✨ 

Confirmation to go live from everyone mentioned:

- [x] @KirstieJane 
- [x] @sgibson91
- [ ] @pherterich
- [x] @louiseabowler
- [x] @GeorgiaHCA <--- Needs to re-review as requested changes
- [x] @gedankenstuecke
- [ ] @yochannah

### CODE REVIEW

**Comment by KirstieJane** [comment]
@gedankenstuecke - what do you think of this blog post? Is it ok? I can add a link to your slides if you'd like, but totally no worries if you're busy!

**Comment by gedankenstuecke** [comment]
Hey! Looks great to me! I can’t upload the PDF of my slides to a more permanent location right now from my phone, but this should work!

[CRI-presentation.pdf](https://github.com/WhitakerLab/whitakerlab.github.io/files/3431466/CRI-presentation.pdf)

**Comment by KirstieJane** [comment]
added your slides @gedankenstuecke - I haven't indicated a license though - want to add one?

**Comment by gedankenstuecke** [comment]
Yeah, I’d be happy to put the slides under CC BY! All the stuff that looks super hand-drawn is actually hand-drawn by me! :D

**Review by sgibson91 — APPROVED** [review]

File: _posts/blog/2019-07-18-lab-meeting.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+**Kirstie** wrote a [little blog post](https://news.sky.com/story/turings-story-is-motivation-to-keep-fighting-for-equity-and-justice-11763751) for Sky News about Alan Turing being on the £50 note.
+
+The piece closes with the following call to action:
+
+> Alan Turing will now become the iconic face of the £50 note.
+> But he also represents everyone who has served their country, those giants of science on whose shoulders we now stand, and all people who have been - and continue to be - denied their human rights.
+> 
+> I look to Alan Turing both as a role model to inspire new and creative research questions and as motivation to keep fighting for equity and justice for all marginalised members of our society.
+> In his own words: **"We can only see a short distance ahead, but we can see plenty there that needs to be done."**
```
Comment (GeorgiaHCA) [inline _posts/blog/2019-07-18-lab-meeting.md:31]: This is such a great quote! I love it

File: _posts/blog/2019-07-18-lab-meeting.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+**Sarah's** got herself into an infinite loop of [building cool bots](https://github.com/HelmUpgradeBot?tab=repositories) to do her work for her 😂
+
+**Patricia** is wrapping up her current job and getting ready for new adventures.
+But before she goes she'll run a "Friendly GitHub" workshop (inspired by [this one](https://kirstiejane.github.io/friendly-github-intro/)) for library colleagues next week.
+
+**Louise** has been learning about scientific file formats and how to improve them by learning from database design expertise.
+Kirstie recommended that Louise (and everyone!) should checkout Saul Pwanson's amazing talk at [csv,conf,v4](https://csvconf.com/) on ["How a File Format Led to a Crossword Scandal"](https://www.youtube.com/watch?v=9aHfK8EUIzg).
+You can see all the csv,conf talks online at this [YouTube playlist](https://www.youtube.com/playlist?list=PLg5zZXwt2ZW72M0KOOt8xQNXsSPMp4PNd).
+
+**Georgia** as built a demo for the Autistica/Citizen science platform!
```
Comment (GeorgiaHCA) [inline _posts/blog/2019-07-18-lab-meeting.md]: Tiny Typo to fix - "as" to "has"

**Review by GeorgiaHCA — CHANGES_REQUESTED** [review]
Heya Kirstie! This looks great thanks for writing it :-). Just noticed one tiny typo.

**Review by LouiseABowler — APPROVED** [review]
Looks good to me!

File: _posts/blog/2019-07-18-lab-meeting.md (hunk 2 above)
Comment (KirstieJane) [reply _posts/blog/2019-07-18-lab-meeting.md]: Thank you @GeorgiaHCA!! Fixed now.

Please can you re-review and approve when you have time 😄

**Review by GeorgiaHCA — APPROVED** [review]
Looks good! ✨

**Pull request merged**